JavaFXUserInterface/src/main/java/controllers/AssetInfoController.java

```diff
@@ -231,6 +246,17 @@ public void attachEvents() {
             rawDataListPane.getChildren().clear();
             generateRawDataTable();
         });
+
+        assetSaveBtn.setOnMouseClicked(mouseEvent -> {
```

Comment:
The save button could be disabled as long as any of the textfield hasn't been edited, to keep it consistent with other save buttons in our app.

Reply:
I agree with the point, simply because it has to be the same across the app.